activités.md

```diff
@@ -0,0 +1,12 @@
+# **L'activité de serveur watch together**
+
+### **Que sont les activités de serveurs ?**
+
+La fonctionnalité Discord Watch Together permet aux utilisateurs d’ajouter des vidéos YouTube par lien ou de saisir le nom d’un créateur ou d’une vidéo pour rechercher sur YouTube des vidéos pertinentes.
+
+### **Comment activer l'intégration watch together ?**
+
+Pour commencer à regarder YouTube avec l'intégration il faut avoir un bot Discord possédant la fonctionnalité watch together, ainsi il faudra lancer l'activité dans un salon vocal uniquement, ensuite les utilisateurs dans le vocal doivent accepter la confidentialité et les conditions d’utilisation de Youtube et l'activité pourra commencer en cherchant le nom de la vidéo YouTube dans la barre de recherche.
```

Comment:
Tu pourrais séparer la phrase "Pour commencer à regarder YouTube avec l'intégration il faut avoir un bot Discord possédant la fonctionnalité watch together" de "ainsi il faudra lancer l'activité dans un salon vocal uniquement" par un point pour ne pas avoir une phrase excessivement longue et par ailleurs peut-être aussi lister la procédure avec une liste à puce pour rendre celle-ci plus intéressante.

Reply:
Il faudrait aussi peut-être ajouter un lien d'invitation du bot officiel créé par Discord, qui permet l'utilisation de ces jeux in-vocal.

Reply:
Ce serait une bonne idée, effectivement.